Composer/packages/client/src/pages/design/DebugPanel/TabExtensions/DiagnosticsTab/DiagnosticList.tsx

```diff
@@ -87,11 +88,25 @@ function onRenderDetailsHeader(props, defaultRender) {
   );
 }
 
+const BotNameRender: React.FC<{ item: IDiagnosticInfo }> = ({ item }) => {
+  const botName = useRecoilValue(botDisplayNameState(item.projectId));
+  return (
+    <div data-is-focusable css={tableCell}>
+      <div aria-label={formatMessage(`Bot is {botName}`, { botName })} css={content} tabIndex={-1}>
+        {botName}
+      </div>
+    </div>
+  );
+};
+
 export const DiagnosticList: React.FC<IDiagnosticListProps> = ({ diagnosticItems }) => {
   const setExportSkillModalInfo = useSetRecoilState(exportSkillModalInfoState);
-  const projectId = useRecoilValue(currentProjectIdState);
-  const botName = useRecoilValue(botDisplayNameState(projectId));
-  const columns: IColumn[] = [
+  const skillStates = useRecoilValue(skillsStateSelector);
+
+  const getSkillName = (projectId: string) =>
```

Comment:
Dont we have the botName inside the skillStateSelector? Why use the skillNameIdentifier?

Reply:
as we discussed, `botProjectSpace` is a better choice.